### PR TITLE
Fix/45 pass currencies explicitly

### LINKED ISSUE #45: `check_missing_currency` fails if `currencies` is not on search path
https://github.com/RMI-PACTA/pacta.portfolio.audit/blob/d2218000fb395de3f0f3766996de11e033f66211/R/check_missing_currency.R#L7

In this line, `currencies` object is referenced without definition or passing as argument to function.

```
Error in `mutate()`:
ℹ In argument: `has_currency = case_when(...)`.
Caused by error in `case_when()`:
! Failed to evaluate the left-hand side of formula 3.
Caused by error:
! object 'currencies' not found
Backtrace:
     ▆
  1. ├─workflow.pacta:::run_pacta()
  2. │ └─workflow.pacta:::run_audit(cfg_path)
  3. │   └─pacta.portfolio.audit::add_portfolio_flags(portfolio)
  4. │     └─pacta.portfolio.audit:::check_missing_currency(portfolio)
  5. │       └─portfolio_total %>% ...
  6. ├─dplyr::mutate(...)
  7. ├─dplyr:::mutate.data.frame(...)
  8. │ └─dplyr:::mutate_cols(.data, dplyr_quosures(...), by)
  9. │   ├─base::withCallingHandlers(...)
 10. │   └─dplyr:::mutate_col(dots[[i]], data, mask, new_columns)
 11. │     └─mask$eval_all_mutate(quo)
 12. │       └─dplyr (local) eval()
 13. ├─dplyr::case_when(...)
 14. │ └─dplyr:::case_formula_evaluate(...)
 15. │   ├─base::withCallingHandlers(...)
 16. │   └─rlang::eval_tidy(pair$lhs, env = default_env)
 17. ├─currency %in% currencies$currency
 18. └─base::.handleSimpleError(...)
 19.   └─dplyr (local) h(simpleError(msg, call))
 20.     └─rlang::abort(message, parent = cnd, call = error_call)
Execution halted
```

### PR DESCRIPTION
**Pull request by AlexAxthelm**
Pass `currencies` object explicitly to `check_missing_currency()`

closes #45

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: pacta.portfolio.audit
 Title: Functions for Calculating Audit Statistics for PACTA on Capital Transition Monitor
-Version: 0.0.1.9000
+Version: 0.0.1.9001
```
Comment (cjyetman) [inline DESCRIPTION:3]:
```suggestion
Version: 0.0.1.9000
```

File: R/add_portfolio_flags.R [hunk 2]
```diff
@@ -5,16 +5,21 @@
 #' financial data.
 #'
 #' @param portfolio A data frame containing a portfolio
+#' @param currencies A data frame containing a currencies and their exchange
```
Comment (cjyetman) [inline R/add_portfolio_flags.R:8]:
```suggestion
#' @param currencies A data frame containing ISO 4217 currency codes and their exchange
```

**Review by cjyetman — CHANGES_REQUESTED** [review]
looks good with some minor changes

**Comment by cjyetman** [comment]
@AlexAxthelm can you rebuild the docs please... I think the change I suggested probably causes a need for rebuild

**Comment by AlexAxthelm** [comment]
rendered.

**Review by cjyetman — APPROVED** [review]

**Pull request merged**